app/Renderers/CalendarRenderer.php

```diff
@@ -87,6 +87,12 @@ class CalendarRenderer
      */
     protected array $eventEnd = [];
 
+    /**
+     * End date of events displayed on the calendar view
+     * @var bool
+     */
+    protected bool $eventEndsThisMonth = false;
```

Comment:
Not sure why this is a variable on the object when it's only used in a single function.